### DISCUSSION TITLE
Open Draw IO multiple users terminal server

### DISCUSSION THREAD

**Discussion by gcon23** [jgraph/drawio-desktop#991] [Q&A] [answered]
Hello,

We install Draw.io on a terminal server. Now we can only open Draw IO with one user. When another user open Draw IO on the same server, nothing happens. In task manager, I see the draw.io process appear and leave within seconds in the second user session. We don't have this problem with version 16.5.1 only with version 17/18. 

Is there a solutions for this problem?

**Comment by m-mohamedin** [comment, marked as answer]
This bug in Electron is causing the issue
https://github.com/electron/electron/issues/33975

**Reply by gcon23** [reply]
Thank you!

**Reply by gcon23** [reply]
Hello, I read on [https://github.com/electron/electron/issues/33975](url) that the problem is fixed. But how do I know it's also fixed in Draw IO?

**Reply by m-mohamedin** [reply]
We're using the latest electron now, can you please try and confirm?

**Reply by gcon23** [reply]
Hello, i have installed version 18.0.6 and the problem is solved.
Thank you!
